packages/web/.gitignore

```diff
@@ -0,0 +1,41 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
@Fruktus 
Once merged - we need to make sure this does not messes with your code (meaning that I am not taking out files and folders that you need in the main repo)
If so - I will need to move some of the items here to the main .gitIgnore

Reply:
AFAIK this looks ok